src/State/Location.cpp

```diff
@@ -413,7 +413,7 @@ void Location::think()
     if (_scrollTicks + 10 < SDL_GetTicks())
     {
         _scrollTicks = SDL_GetTicks();
-        unsigned int scrollDelta = 5;
+        unsigned int scrollDelta = 10;
 
```

Comment:
This must be placed in different commit. Using git branches for every pull request can do the job